README.md

```diff
@@ -74,6 +74,9 @@ $ cd deeplearnjs
 $ npm run prep # Installs node modules and bower components.
 ```
 
+We recommend using [Visual Studio Code](https://code.visualstudio.com/) for
+development. Make sure to install `clang-format` for IDE auto-formatting.
```

Comment:
add link to https://marketplace.visualstudio.com/items?itemName=xaver.clang-format and mention that it's an *extension*